designs/waralee/src/mini.mjs

```diff
@@ -4,44 +4,48 @@ import { pantsProto } from './pantsproto.mjs'
 // To keep you from printing it completely, you could print this part in paperless mode
 // and only have a single sheet with all the dimensions on it.
 
-function waraleeMini({ options, Path, points, paths, complete, sa, macro, store, part }) {
-  let mini = options.minimizer
-  let separateWaistband = options.separateWaistband
-  if ('waistband' == options.frontPocketStyle) {
-    separateWaistband = true
-  }
-
-  for (var p in points) {
-    points[p].x = points[p].x / mini
-    points[p].y = points[p].y / mini
-  }
-
-  paths.waistFoldBack = paths.waistBack
-    .offset((-1 * store.get('waistBand')) / mini)
-    .attr('class', 'fabric stroke-sm')
-  paths.waistFoldFront = paths.waistFront
-    .offset((-1 * store.get('waistBand')) / mini)
-    .attr('class', 'fabric stroke-sm')
-
-  paths.frontFold = paths.front
-    .offset((-1 * store.get('hem')) / mini)
-    .attr('class', 'fabric stroke-sm')
-  paths.legFold = paths.leg.offset((-1 * store.get('hem')) / mini).attr('class', 'fabric stroke-sm')
-  paths.backFold = paths.back
-    .offset((-1 * store.get('hem')) / mini)
-    .attr('class', 'fabric stroke-sm')
-
-  paths.cutOut = new Path()
-    .move(separateWaistband ? points.bWaistSideSeam : points.bWaistSide)
-    .line(points.mWaist2)
-    .line(points.mWaist1)
-    .line(separateWaistband ? points.fWaistSideSeam : points.fWaistSide)
-    .attr('class', 'help')
-
-  paths.seam.unhide()
-
-  // Complete?
-  if (complete) {
+export const mini = {
+  name: 'waralee.mini',
+  from: pantsProto,
+  draft: ({ options, Path, points, paths, sa, macro, store, expand, part }) => {
+    if (expand) {
+      return part.hide()
+    }
+
+    const mini = points.bWaistBackOverlapSeam.dist(points.fWaistFrontOverlapSeam) / 150
+    const separateWaistband = options.separateWaistband || 'waistband' == options.frontPocketStyle
+
+    for (const p in points) {
+      points[p].x = points[p].x / mini
+      points[p].y = points[p].y / mini
+    }
+
+    paths.waistFoldBack = paths.waistBack
+      .offset((-1 * store.get('waistBand')) / mini)
+      .attr('class', 'fabric stroke-sm')
+    paths.waistFoldFront = paths.waistFront
+      .offset((-1 * store.get('waistBand')) / mini)
+      .attr('class', 'fabric stroke-sm')
```

Comment:
Tip: We have a better API for this now:

```js
.addClass('fabric stroke-sm')
```

See: https://freesewing.dev/reference/api/path/addclass